### DISCUSSION TITLE
macaroon files not created automatically

### DISCUSSION THREAD

**Discussion by RyanPaul31** [lightninglabs/lightning-terminal#660] [Troubleshooting]
Hi Team,

I have followed the same procedure for Lit configurations and It was running. But the problem is macaroon files are not automatically created inside .lit folder and also tls.cert and tls.key files are also not generated.

Also macaroon files are not generated inside any of .loop, .pool and .faraday folders.

Kindly guide me where it went wrong. What needs to be done to solve this issues?

Thanks in Advance!

**Comment by guggero** [comment]
That sounds like Lit isn't starting up properly. Can you post some logs please?

**Comment by RyanPaul31** [comment]
LTND: We're not running within systemd or the service type is not 'notify'
LTND: Waiting for wallet encryption password. Use `lncli create` to create a wallet, `lncli unlock` to unlock an existing wallet, or `lncli changepassword` to change the password of an existing wallet and unlock it.
LTND: Received interrupt
LTND: Shutting down...
LTND: Gracefully shutting down.
LTND: unable to set up wallet password listeners: shutting down
LTND: Shutting down because error in main method: error creating wallet config: unable to set up wallet password listeners: shutting down
STAT: Setting the lit sub-server as errored: could not start Lit: %v
STAT: could not start the lit sub-server: could not start Lit: received the shutdown signal
LITD: Shutdown signal received
GRPC: [core] [Channel #83] Channel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #83 SubChannel #84] Subchannel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #42] Channel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #29 SubChannel #30] Subchannel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #29 SubChannel #30] Subchannel deleted
GRPC: [core] [Channel #29] Channel deleted
GRPC: [transport] transport: loopyWriter exited. Closing connection. Err: transport closed by client
GRPC: [transport] transport: closing: connection error: desc = "error reading from server: read tcp 127.0.0.1:52714->127.0.0.1:10009: use of closed network connection"
GRPC: [core] [Channel #77] Channel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #77 SubChannel #78] Subchannel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #77 SubChannel #78] Subchannel deleted
GRPC: [core] [Channel #77] Channel deleted
GRPC: [transport] transport: loopyWriter exited. Closing connection. Err: transport closed by client
GRPC: [transport] transport: closing: connection error: desc = "error reading from server: read tcp 127.0.0.1:52686->127.0.0.1:10009: use of closed network connection"
GRPC: [core] [Channel #42 SubChannel #44] Subchannel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #42 SubChannel #44] Subchannel deleted
GRPC: [core] [Channel #42] Channel deleted
GRPC: [core] [Channel #47 SubChannel #56] Subchannel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #13 SubChannel #14] Subchannel Connectivity change to SHUTDOWN
GRPC: [core] [Channel #79 SubChannel #82] Subchannel deleted
GRPC: [core] [Channel #89] Channel Connectivity change to SHUTDOWN
STAT: Setting the faraday sub-server as stopped
GRPC: [core] [Channel #86] Channel Connectivity change to SHUTDOWN
GRPC: [core] [Server #2 ListenSocket #3] ListenSocket deleted
GRPC: [core] [Channel #38] Channel Connectivity change to SHUTDOWN

**Comment by guggero** [comment]
So you started Lit in integrated mode, which means it is also starting an `lnd` node. You have to create a wallet first (see second line of the log). Once you create the wallet (which doesn't require a macaroon), the other macaroons and certificates will be created.

**Comment by RyanPaul31** [comment]
I've created wallet and macaroon files are created only inside .lnd folder not inside other folders

**Comment by guggero** [comment]
The wallet is probably still syncing, which takes a while. Check the logs for more information or errors.

**Comment by RyanPaul31** [comment]
I'm frequently getting this error only : GRPC: [transport] transport: closing: connection error: desc = "error reading from server: read tcp 127.0.0.1:52656->127.0.0.1:10009: use of closed network connection"

**Comment by guggero** [comment]
That just means a component is trying to connect to lnd. But without any context that could be anything.

**Comment by RyanPaul31** [comment]
How can I proceed further?

**Comment by guggero** [comment]
Give us more info, like the log output since you created the wallet.
Going to move this into a discussion, as it's related to trouble shooting.

**Comment by RyanPaul31** [comment]
Mostly I'm getting this error

GRPC: [transport] transport: loopyWriter exited. Closing connection. Err: transport closed by client
GRPC: [transport] transport: closing: connection error: desc = "error reading from server: read tcp 127.0.0.1:52686->127.0.0.1:10009: use of closed network connection"

**Comment by RyanPaul31** [comment]
Can anyone please help me out to solve this issue?

**Reply by guggero** [reply]
If you post more of the log, we might be able to.

**Comment by RyanPaul31** [comment]
I'm getting response like this in my log file other than those errors

[DBG] LNWL: Skipping coinbase tx cc475e838bf645b1fdfa6bf2a1f10b08f1ba0ba53a24babde6542c0091fa72e3
[DBG] LNWL: Skipping coinbase tx 88fe240e0f23db78c62c2a681f6ddb18f5f60e63d245feb81bf16a33efefd373
[DBG] LNWL: Skipping coinbase tx e9e9bbeed7e04351d0ed91afbabb0ba238506c573abb6ec8a93212cd7fe6acf2
[DBG] LNWL: Skipping coinbase tx e45d3b3a67b4baddf59a8fdbeb9e07cadee843ee1919b0dd64b7e33abddadbfe
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LNWL: Skipping coinbase tx 03f239d095b488afc610c19e478c612768dd5bdb08e8fb4c7e5396c1ee73e92a
[DBG] LNWL: Skipping coinbase tx c64d20d2d29907b65c5d22ceb144cb7043ea34e6f8db727df4b73a24f2df8c2f
[DBG] LNWL: Skipping coinbase tx f5b02e7e490b917ec75aa5ac843f577e3bf923ac609061c3e6463927a6f21a4e
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] RPCS: [/lnrpc.Lightning/GetInfo] requested
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] RPCS: [/lnrpc.Lightning/GetInfo] requested
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] RPCS: [/lnrpc.Lightning/GetInfo] requested
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] RPCS: [/lnrpc.Lightning/GetInfo] requested
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LTND: Syncing to block timestamp: 2017-12-10 01:52:45 +0000 UTC, is synced=false
[DBG] LNWL: Skipping coinbase tx 3eb0c301ecf9f1748493582a7d7c1c5fe6fb197b88f729e24a944b565bd3146e
[DBG] LTND: Syncing to block timestamp: 2017-12-10 02:36:28 +0000 UTC, is synced=false

**Reply by guggero** [reply]
That means the wallet is still syncing to the chain. The macaroon files should be there now though. You'll need to wait until the `is synced=false` messages go away.

**Reply by RyanPaul31** [reply]
@guggero  As you mentioned, I've waited until the synced=false message to go away. I've got macaroon files now but still tls.cert tls.key files are not generated

**Reply by guggero** [reply]
What directories are you looking in? The `lnd` certificate and key files should be generated at that point. Then the other services within Lightning Terminal should start up. Are there any other log messages after the sync is complete?

**Reply by RyanPaul31** [reply]
I'm checking inside .lit folder. I can see macaroon file inside .lit, .loop, .pool, .faraday folders. I don't see tls.cert file only in .lnd folder.
My log looks like:

[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelEdgePolicy, scid=792255302528860161
[DBG] CRTR: Received ChannelEdgePolicy for channel 792255302528860161
[DBG] DISC: Validating ChannelUpdate: channel=792233312226181121, from node=03cde60a6323f7122d5178255766e38114b4722ede08f7c9e0c5df9b912cc201d6, has edge=false
[DBG] DISC: Processing NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2023-05-25 14:08:34 +0000 UTC, node=0294fd62bc6f4ad2804dfb01237baaf8b8dd5646cf60639e6586c5a1d9d4ed01f4
[DBG] DISC: Processed ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=791125004570132481, timestamp=2023-10-21 18:09:26 +0000 UTC
[DBG] PEER: Peer(025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5): Received NodeAnnouncement(node=03ef74233102882aead89fe1dbbced341a071486679b62416f0e84c532585d917a, update_time=2022-03-25 00:28:13 +0000 UTC) from 025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735
[DBG] CRTR: Received ChannelEdgePolicy for channel 791186577271095296
[DBG] DISC: Processed ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=791127203567435777, timestamp=2023-10-23 16:37:21 +0000 UTC
[DBG] CRTR: Waiting for dependent on job=lnwire.NodeAnnouncement, pub=038cd67913dd362f760a1ce86f376d23fa71cf24d06b8790a2a7b71699ea1dc2d8
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelEdgePolicy, scid=792238809859227649
[DBG] CRTR: Received ChannelEdgePolicy for channel 792238809859227649

**Reply by guggero** [reply]
Can you please upload the whole log? This is just debug information. You can attach to the message on GitHub directly.

**Reply by RyanPaul31** [reply]
My entire logs are repeated like this only. Kindly check the below:

[DBG] DISC: Processed ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=818428077248413696, timestamp=2023-10-23 16:36:25 +0000 UTC
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelEdgePolicy, scid=819071291725316097
[DBG] CRTR: Received ChannelEdgePolicy for channel 819071291725316097
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820342327015768064
[DBG] DISC: Processing ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=820343426520514560
[DBG] DISC: Processed ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=818414883121332224, timestamp=2023-10-23 14:05:59 +0000 UTC
[DBG] CRTR: Received ChannelEdgePolicy for channel 818406086996525057
[DBG] CRTR: Received ChannelEdgeInfo for channel 819687018030235649
[DBG] DISC: Finish adding edge for short_chan_id: 819663928352112640
[DBG] DISC: Processed ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352112640
[DBG] DISC: Processed ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819063595106893825, timestamp=2023-10-21 11:23:43 +0000 UTC
[DBG] DISC: Processing NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2023-10-24 06:44:44 +0000 UTC, node=032d330f24cfac7142561e961ba5c11f17ea4dde3e2b3cd257241c0073787e3310
[DBG] CRTR: Checking stale node 303332643333306632346366616337313432353631653936316261356331316631376561346464653365326233636432353732343163303037333738376533333130 got Ignoring outdated announcement for 032d330f24cfac7142561e961ba5c11f17ea4dde3e2b3cd257241c0073787e3310
[DBG] DISC: Skipped processing stale node: 032d330f24cfac7142561e961ba5c11f17ea4dde3e2b3cd257241c0073787e3310
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819666127345811456, 
[DBG] CRTR: process network updates got: Ignoring outdated announcement for 02eab9f172e03bf64c8448866d18584af2f69c5a2de764f741eba25b2a4e80192b
[DBG] CRTR: Received ChannelEdgePolicy for channel 818418181619580929
[DBG] CRTR: Waiting for dependent on job=channeldb.LightningNode, pub=022b0eb6eac43c0b7e56dc8b6e8205f735a80966521f553db56deff197b0035a7f
[DBG] DISC: Validating ChannelUpdate: channel=819666127345811456, from node=032d330f24cfac7142561e961ba5c11f17ea4dde3e2b3cd257241c0073787e3310, has edge=false
[DBG] CRTR: Waiting for dependent on job=lnwire.NodeAnnouncement, pub=0283ca0b7eca022bcf8f32aa418e5ff2bd375806faba40409cdc8e48c2b32ef725
[DBG] DISC: Processed ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819098779404664833, timestamp=2023-10-18 13:43:06 +0000 UTC
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928351326208, 
[DBG] DISC: Validating ChannelUpdate: channel=819663928351326208, from node=02432210f1b3940d8af701c2f2fb4058a4ab93b11150d35c4c4164fbd0b454ff51, has edge=false
[DBG] DISC: Processing NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2023-10-24 10:09:10 +0000 UTC, node=0321c95cb9242b489bc89a80effd391b93fe8fec05cd66f7825b72d6880f1feece
[DBG] DISC: Processing NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2023-09-08 04:18:25 +0000 UTC, node=02432210f1b3940d8af701c2f2fb4058a4ab93b11150d35c4c4164fbd0b454ff51
[DBG] CRTR: Checking stale node 303332316339356362393234326234383962633839613830656666643339316239336665386665633035636436366637383235623732643638383066316665656365 got Ignoring outdated announcement for 0321c95cb9242b489bc89a80effd391b93fe8fec05cd66f7825b72d6880f1feece
[DBG] DISC: Skipped processing stale node: 0321c95cb9242b489bc89a80effd391b93fe8fec05cd66f7825b72d6880f1feece
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819666127345811456, 
[DBG] DISC: Validating ChannelUpdate: channel=819098779404664833, from node=024a8228d764091fce2ed67e1a7404f83e38ea3c7cb42030a2789e73cf3b341365, has edge=false
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352112640, 
[DBG] DISC: Validating ChannelUpdate: channel=819663928352112640, from node=03fc8aaec61b42da62e36f5473623ca9b91285d7af62c6189d14bcf84c549d473b, has edge=false
[DBG] DISC: Processing ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=820342327015768064
[DBG] DISC: Processing NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2023-10-24 02:34:21 +0000 UTC, node=03fc8aaec61b42da62e36f5473623ca9b91285d7af62c6189d14bcf84c549d473b
[DBG] CRTR: Checking stale node 303366633861616563363162343264613632653336663534373336323363613962393132383564376166363263363138396431346263663834633534396434373362 got Ignoring outdated announcement for 03fc8aaec61b42da62e36f5473623ca9b91285d7af62c6189d14bcf84c549d473b
[DBG] DISC: Adding edge for short_chan_id: 820343426520514560
[DBG] DISC: Skipped processing stale node: 03fc8aaec61b42da62e36f5473623ca9b91285d7af62c6189d14bcf84c549d473b
[DBG] CRTR: Received ChannelEdgePolicy for channel 818463261627383809
[DBG] DISC: Processed NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2022-08-09 22:25:16 +0000 UTC, node=039b019225793b02ca7fe17c71b860e3500ed60ed1450592c322057d1b041473cc
[DBG] CRTR: Waiting for dependent on job=lnwire.NodeAnnouncement, pub=03011e480a671ac71dbc3fc3e8fe0db32bb9a10cc4d824663e09557d446ef80679
[DBG] CRTR: Checking stale node 303234333232313066316233393430643861663730316332663266623430353861346162393362313131353064333563346334313634666264306234353466663531 got Ignoring outdated announcement for 02432210f1b3940d8af701c2f2fb4058a4ab93b11150d35c4c4164fbd0b454ff51
[DBG] DISC: Skipped processing stale node: 02432210f1b3940d8af701c2f2fb4058a4ab93b11150d35c4c4164fbd0b454ff51
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928351326208, 
[DBG] CRTR: Received ChannelEdgePolicy for channel 818445669440487424
[DBG] DISC: Adding node: 02eab9f172e03bf64c8448866d18584af2f69c5a2de764f741eba25b2a4e80192b got error: Ignoring outdated announcement for 02eab9f172e03bf64c8448866d18584af2f69c5a2de764f741eba25b2a4e80192b
[DBG] CRTR: Received ChannelEdgeInfo for channel 819689217054867457
[DBG] CRTR: Received ChannelEdgeInfo for channel 819667226892435457
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352112640, 
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820343426520514560
[DBG] DISC: Processed ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=818406087051051009, timestamp=2023-08-04 18:43:04 +0000 UTC
[DBG] DISC: Processing ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=820347824647700481
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820347824647700481
[DBG] DISC: Processed NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2023-10-23 16:43:06 +0000 UTC, node=0284feb87c565027219fbd40137805469537b298db83747f398322293a8d283105
[DBG] CRTR: Received ChannelEdgeInfo for channel 819689217076166657
[DBG] CRTR: Waiting for dependent on job=lnwire.NodeAnnouncement, pub=03a492d3f8f52602a775b87507b82965de8d3e1a9cc282d629f474870a8c90d886
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelEdgePolicy, scid=819137262262222849
[DBG] CRTR: Received ChannelEdgePolicy for channel 819137262262222849
[DBG] DISC: Validating ChannelUpdate: channel=819063595106893825, from node=03df3f0a2fd6bea5429a596461ce784c922b2981ada1af89cfefcd9ccfb16c16a7, has edge=false
[DBG] CRTR: Waiting for dependent on job=lnwire.NodeAnnouncement, pub=02b2cb2f6fa89b8e204cd5a0bad82edf975e2132e80d1239d510055bfe6ac095c6
[DBG] DISC: Processing ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=820347824623124480
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820347824623124480
[DBG] DISC: Finish adding edge for short_chan_id: 819663928352047104
[DBG] DISC: Processed ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352047104
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelEdgePolicy, scid=819130665253732353
[DBG] CRTR: Received ChannelEdgePolicy for channel 819130665253732353
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352047104, 
[DBG] DISC: Validating ChannelUpdate: channel=819663928352047104, from node=036d306e01a65128e5130c99a004119f36bc296ee6f002a7c3c478530b8249a548, has edge=false
[DBG] DISC: Adding edge for short_chan_id: 820347824647700481
[DBG] DISC: Processing NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2023-09-14 15:15:06 +0000 UTC, node=02e9046555a9665145b0dbd7f135744598418df7d61d3660659641886ef1274844
[DBG] CRTR: Checking stale node 303265393034363535356139363635313435623064626437663133353734343539383431386466376436316433363630363539363431383836656631323734383434 got Ignoring outdated announcement for 02e9046555a9665145b0dbd7f135744598418df7d61d3660659641886ef1274844
[DBG] DISC: Skipped processing stale node: 02e9046555a9665145b0dbd7f135744598418df7d61d3660659641886ef1274844
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352047104, 
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820347824647700481
[DBG] DISC: Finish adding edge for short_chan_id: 819662828911788033
[DBG] DISC: Processed ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819662828911788033
[DBG] DISC: Processing ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=820350023601946625
[DBG] DISC: Processing NodeAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, timestamp=2023-10-23 16:28:37 +0000 UTC, node=02de223c6d19fce250030f9569dc1dfbf3221987cbce358a060ad83e8ba5149242
[DBG] CRTR: Checking stale node 303264653232336336643139666365323530303330663935363964633164666266333232313938376362636533353861303630616438336538626135313439323432 got Ignoring outdated announcement for 02de223c6d19fce250030f9569dc1dfbf3221987cbce358a060ad83e8ba5149242
[DBG] DISC: Skipped processing stale node: 02de223c6d19fce250030f9569dc1dfbf3221987cbce358a060ad83e8ba5149242
[DBG] CRTR: Received ChannelEdgeInfo for channel 819710107908833281
[DBG] DISC: Finish adding edge for short_chan_id: 819663928352702464
[DBG] DISC: Processed ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352702464
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819662828911788033, 
[DBG] DISC: Adding edge for short_chan_id: 820347824623124480
[DBG] DISC: Validating ChannelUpdate: channel=819662828911788033, from node=02de223c6d19fce250030f9569dc1dfbf3221987cbce358a060ad83e8ba5149242, has edge=false
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352702464, 
[DBG] DISC: Validating ChannelUpdate: channel=819663928352702464, from node=0242a4ae0c5bef18048fbecf995094b74bfb0f7391418d71ed394784373f41e4f3, has edge=false
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820350023601946625
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820347824623124480
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820350023601946625
[DBG] CRTR: Received ChannelEdgePolicy for channel 818450067527303169
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820343426520514560
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820347824637673472
[DBG] CRTR: Waiting for dependent on job=lnwire.ChannelUpdate, scid=820347824637673472
[DBG] DISC: Adding edge for short_chan_id: 820342327015768064
[DBG] CRTR: Waiting for dependent on job=lnwire.NodeAnnouncement, pub=03ba0dc26e137f7fc5406908aaf614807cde5010a56b39973d68377fb0aa77e5d5
[DBG] DISC: Processing ChannelAnnouncement: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=820347824637673472
[DBG] DISC: Adding edge for short_chan_id: 820350023601946625
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819663928352702464, 
[DBG] DISC: Processing ChannelUpdate: peer=025f1456582e70c4c06b61d5c8ed3ce229e6d0db538be337a2dc6d163b0ebc05a5@52.86.210.65:9735, short_chan_id=819662828911788033, 
[DBG] DISC: Adding edge for short_chan_id: 820347824637673472

**Reply by guggero** [reply]
Please do another restart and then upload the complete file, as a file, not pasting text.

**Reply by RyanPaul31** [reply]
Can you please guide me to set correct lnd.conf and lit.conf file? I think there may be the reason for these errors

**Reply by guggero** [reply]
I guess the first question to ask is: Do you want to run `lnd` as part of the whole `litd` bundle (`--lnd-mode=integrated`)? Or do you want to run that as a separate process (`--lnd-mode=remote`)? There are pros and cons for both models, but assuming you aren't yet running an `lnd` node it probably makes sense to run everything in a single process. If you do that, then you only need `lit.conf` (no `lnd.conf`), which is described here: https://github.com/lightninglabs/lightning-terminal/blob/master/doc/config-lnd-integrated.md

If you are looking to set up `litd` with an existing `lnd` node, here's the doc for that: https://github.com/lightninglabs/lightning-terminal/blob/master/doc/config-lnd-remote.md (in that case you need both an `lnd.conf` and a `lit.conf` file).

**Reply by RyanPaul31** [reply]
I've started integrating lit by using (--lnd-mode=remote). I almost got the result but I got errors mentioned in the attachment. I got the message like "Waiting for lnd to be fully synced to its chain backend, this might take a while" in loop, pool, faraday and lit logs . In lnd log, I'm getting logs like:

LNWL: Opened wallet
CHRE: Primary chain is set to: bitcoin
CHRE: Initializing bitcoind backed fee estimator in ECONOMICAL mode
LNWL: Started listening for bitcoind transaction notifications via ZMQ on 127.0.0.1:28333
LNWL: Started listening for bitcoind block notifications via ZMQ on 127.0.0.1:28332
LNWL: Started polling mempool to cache new transactions
LNWL: The wallet has been unlocked without a time limit
CHRE: LightningWallet opened
HSWC: Cleaning circuits from disk for closed channels
HSWC: Finished cleaning: no closed channels found, no actions taken.
HSWC: Restoring in-memory circuit state from disk
HSWC: Payment circuits loaded: num_pending=0, num_open=0
LTND: Channel backup proxy channel notifier starting
ATPL: Instantiating autopilot with active=false, max_channels=5, allocation=0.600000, min_chan_size=20000, max_chan_size=16777215, private=false, min_confs=1, conf_target=3
LTND: We're not running within systemd or the service type is not 'notify'
LTND: Waiting for chain backend to finish sync, start_height=600889
RPCS: New transaction subscription
RPCS: Canceling transaction subscription
RPCS: New transaction subscription
RPCS: Canceling transaction subscription

Kindly help me to fix this. Thanks in advance!

![image](https://github.com/lightninglabs/lightning-terminal/assets/107464851/ba8590cb-a8c3-4fe5-829b-db67a994e451)

**Reply by guggero** [reply]
> LTND: Waiting for chain backend to finish sync, start_height=600889

Your `lnd` is still syncing.

**Reply by RyanPaul31** [reply]
Its working once lnd synced. Thank you so much for your support @guggero
